experiment/contracts/CheckpointRepo.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity >=0.4.21 <0.6.0;
+
+contract CheckpointRepo {
```

Comment:
It feels easy to add a federation of 3 and this would make the claims in our paper seem much more plausible without too much effort. Otherwise we give the reviewer an easy way out to reject.